src/app/portlets/dot-edit-content/dot-edit-content.component.ts

```diff
@@ -31,6 +33,8 @@ export class DotEditContentComponent implements OnInit {
 
     constructor(
         private dotConfirmationService: DotConfirmationService,
+        private dotMenuService: DotMenuService,
+        private ref: ChangeDetectorRef,
```

Comment:
Why do you need the 'ChangeDetectorRef'?

Reply:
Removed.